src/mlbstandings/web.py

```
@@ -1,17 +1,27 @@
 import backoff
 import requests
-
+# TODO remove once we're running on a system that works with urllib3 + baseball-reference.com
+import urllib.request # import Request, urlopen
+import urllib.error
+ 
 class Web:
     @staticmethod
+    def retryable(code: int) -> bool:
```

Ruff (0.11.9)

Unused static method argument: `code`

(ARG004)